src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -68,11 +68,11 @@ object DepositHandler {
     val tempDir = new File(SwordProps("tempdir"), id)
 
     val result = for {
-      _        <- checkBaseDir()
+      _        <- checkBagStoreBaseDir()
```

Comment:
Changed the name of the function to avoid confusion with anything in `easy-sword2` itself that could be considered a base directory (such as the deposits directory).